lbrynet/daemon/Downloader.py

```diff
@@ -150,6 +152,12 @@ def finish(self, results, name):
         self._check_status(status)
         defer.returnValue(self.download_path)
 
```

Comment:
I think this fail() function just needs to stop the looping call and reraise the err, 
It's attached as an errback to self.downloader.start(), (downloader is an ManagedEncryptedFileDownloader, it doesn't raise DownloadDataTimeout)  , I think you probablly mistook this as being attached as an errback to Downloader.start()

Reply:
changed it to this